File: algorithm/src/main/java/baekjoon/p01748/Main.java
package baekjoon.p01748;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.nextInt()));
    }

    private static int solve(int n) {
        int ret = 0;
        for (int i = 1; i <= n; i *= 10) {
            ret += (n - i + 1);
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p01748/Main.kt
package baekjoon.p01748

fun main(args: Array<String>) {
    println(solve(readLine()!!.toInt()))
}

private fun solve(n: Int): Int {
    var ret = 0
    var i = 1

    while (i <= n) {
        ret += (n - i + 1)
        i *= 10
    }

    return ret
}